recipes/h5py/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+export HDF5_DIR="${PREFIX}"
+
+"${PYTHON}" setup.py install
```

Comment:
I thin that [h5py](https://github.com/h5py/h5py/blob/master/setup.py#L12) can use setuptools so we can use ` --single-version-externally-managed  --record record.txt`